style.css

```diff
@@ -1,34 +1,104 @@
 * {
   background-color: #fff;
-  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 18px;
 }
 
-.center {
+.container-nav-menu {
   display: flex;
-  flex-direction: column;
-  justify-content: center;
+  justify-content: space-around;
   align-items: center;
-  padding-top: 60px;
-}
-
-.button {
   border: 2px solid black;
-  width: 60px;
-  float: right;
 }
 
 .container {
+  padding: 5px;
   border: 2px solid black;
-  width: 50%;
+}
+
+.logo {
+  font-size: 40px;
+  font-weight: bold;
+}
+
+.menu__items {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.menu__items li {
+  list-style-type: none;
+  padding: 20px;
+}
+
+.middle-link {
+  border-left: 2px solid black;
```

Comment:
- [ ] [OPTIONAL]
It will be a nice feature to add; to improve upon the UX of the navigation menu items, thus, each menu item should show an indication that the corresponding page is active. For instance, when the _Add New_ menu item is clicked, it should change color to indicate that the corresponding page is active.

**Showing active menu items:**
![image](https://user-images.githubusercontent.com/12745474/234540359-15febd7e-d0ce-4053-8ee2-3a4304bd5c0e.png)